templates/common/members/userinfo.html

```diff
@@ -20,7 +20,7 @@
                 <!-- User Information Section -->
                 <h3>{{ user.username }}</h3>
                 <div id="userInfo">
-                    <table class="table table-dark table-striped">
+                    <table id="table" class="table table-striped">
```

Comment:
Would have preferred to use classes for this but for some reason they were not being applied to DaTe.